ontology/ontospecies/OntoSpecies_v2.owl

```diff
@@ -2506,7 +2516,7 @@
 
     <owl:Class rdf:about="http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#Density">
         <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#ThermoProperty"/>
-        <rdfs:comment>Density (with unit) and specific gravity (without unit) of a compound. Density is mass of a unit volume of a compound and commonly expressed in units of kg/m3 or g/cm3. Specific gravity, also known as relative density, is a unit-less quantity, defined as the ratio of the density of a compound to that of a standard reference material (typically, water at 4 °C for liquids and air at room temperature [20 °C or 68 °F] for gases).</rdfs:comment>
+        <rdfs:comment>Density (with unit) and specific gravity (without unit) of a compound. Density is mass of a unit volume of a compound and commonly expressed in units of kg/m3 or g/cm3. Specific gravity, also known as relative density, is a unit-less quantity, defined as the ratio of the density of a compound to that of a standard reference material (typically, water at 4 �C for liquids and air at room temperature [20 �C or 68 �F] for gases).</rdfs:comment>
```

Comment:
The ° (degree) symbol is not rendered correctly on line 2519. You can use the following comment instead:

<rdfs:comment>Density (with unit) and specific gravity (without unit) of a compound. Density is mass of a unit volume of a compound and commonly expressed in units of kg/m3 or g/cm3. Specific gravity, also known as relative density, is a unit-less quantity, defined as the ratio of the density of a compound to that of a standard reference material (typically, water at 4&nbsp;&deg;C for liquids and air at room temperature [20&nbsp;&deg;C or 68&nbsp;&deg;F] for gases).</rdfs:comment>

Reply:
Addressed in commits [3498519](https://github.com/TheWorldAvatar/ontology/pull/4/commits/3498519384de39842d5f2b5ea998123c1ea0ace7) and [2d5b674](https://github.com/TheWorldAvatar/ontology/pull/4/commits/2d5b67467e07e0fe5fbde3cc371679a27ece9fcc).